python/pyspark/sql/functions.py

```diff
@@ -6784,7 +6784,7 @@ def date_format(date: "ColumnOrName", format: str) -> Column:
     Examples
     --------
     >>> df = spark.createDataFrame([('2015-04-08',)], ['dt'])
-    >>> df.select(date_format('dt', 'MM/dd/yyy').alias('date')).collect()
+    >>> df.select(date_format('dt', 'MM/dd/yyyy').alias('date')).collect()
```

Comment:
This might be a type but not a bug because 3-y means minimum number not exact number.

Reply:
We should run all examples, and as you can see the output is correct.